wandb/apis/public.py

```diff
@@ -65,6 +65,7 @@
 if TYPE_CHECKING:
     import wandb.apis.reports
     import wandb.apis.reports.util
+    from wandb import Artifact as LocalArtifact
```

Comment:
I like the "draft/committed" terminology better; can we standardize on calling this a DraftArtifact?

Reply:
The current terminology is "local" / "public". I'd like to stay consistent - if we change it, we should change it everywhere at once. Given that I'm merging the classes, it probably doesn't make sense to do now.